administration/src/project-configs/nuernberg/__tests__/csvExport.test.tsx

```diff
@@ -0,0 +1,54 @@
+//import { stringify } from 'csv-stringify/browser/esm/sync'
+import CardBlueprint from '../../../cards/CardBlueprint'
+import { CreateCardsResult } from '../../../cards/createCards'
+import { DynamicActivationCode, StaticVerificationCode } from '../../../generated/card_pb'
+import { Region } from '../../../generated/graphql'
+import getProjectConfig from '../../getProjectConfig'
+import nuernbergConfig from '../config'
+import config from '../config'
+import { buildCsvLine } from '../csvExport'
+
+jest.mock('csv-stringify/browser/esm/sync', () => ({
+  stringify: (input: any[]) => input[0].join(',')
+}))
```

Comment:
@ztefanie like this it works. I have no idea why wrapping it in `jest.fn()` does not work... apparently `jest.fn()` does not return what it should...

This should definitely work (but doesn't):
```suggestion
jest.mock('csv-stringify/browser/esm/sync', () => ({
  stringify: jest.fn((input: any[]) => input[0].join(','))
}))
```

If you need to change the implementation of the mock, you can also do:
```suggestion
jest.mock('csv-stringify/browser/esm/sync', () => ({
  stringify: jest.fn()
}))
```
and then further below:
```ts
mocked(stringify).mockImplementation((input: any[]) => input[0].join(',')) 
```

Reply:
Why do we have to mock it in the first place? Because its an esm module?

Reply:
Thanks @steffenkleinle 
Yes we need to mock it because it is an esm module.